packages/features/ee/organizations/pages/settings/appearance.tsx

```diff
@@ -0,0 +1,241 @@
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { MembershipRole } from "@calcom/prisma/enums";
+import { trpc } from "@calcom/trpc/react";
+import {
+  Button,
+  ColorPicker,
+  Form,
+  Meta,
+  showToast,
+  SkeletonButton,
+  SkeletonContainer,
+  SkeletonText,
+  Switch,
+} from "@calcom/ui";
+
+import ThemeLabel from "../../../../settings/ThemeLabel";
+import { getLayout } from "../../../../settings/layouts/SettingsLayout";
+
+const SkeletonLoader = ({ title, description }: { title: string; description: string }) => {
+  return (
+    <SkeletonContainer>
+      <Meta title={title} description={description} />
+      <div className="mt-6 mb-8 space-y-6">
+        <div className="flex items-center">
+          <SkeletonButton className="mr-6 h-32 w-48 rounded-md p-5" />
+          <SkeletonButton className="mr-6 h-32 w-48 rounded-md p-5" />
+          <SkeletonButton className="mr-6 h-32 w-48 rounded-md p-5" />
+        </div>
+        <div className="flex justify-between">
+          <SkeletonText className="h-8 w-1/3" />
+          <SkeletonText className="h-8 w-1/3" />
+        </div>
+
+        <SkeletonText className="h-8 w-full" />
+
+        <SkeletonButton className="mr-6 h-8 w-20 rounded-md p-5" />
+      </div>
+    </SkeletonContainer>
+  );
+};
+
+interface OrgAppearanceValues {
+  hideBranding: boolean;
+  hideBookATeamMember: boolean;
+  brandColor: string;
+  darkBrandColor: string;
+  theme: string | null | undefined;
+}
+
+const OrgAppearanceView = () => {
+  const { t } = useLocale();
+  const router = useRouter();
+  const utils = trpc.useContext();
+
+  const mutation = trpc.viewer.organizations.update.useMutation({
+    onError: (err) => {
+      showToast(err.message, "error");
+    },
+    async onSuccess() {
+      await utils.viewer.teams.get.invalidate();
+      showToast(t("your_team_updated_successfully"), "success");
+    },
+  });
+
+  const { data: currentOrg, isLoading } = trpc.viewer.organizations.listCurrent.useQuery(undefined, {
+    onError: () => {
+      router.push("/settings");
+    },
+  });
+
+  const form = useForm<OrgAppearanceValues>({
+    defaultValues: {
+      theme: currentOrg?.theme,
+      brandColor: currentOrg?.brandColor,
+      darkBrandColor: currentOrg?.darkBrandColor,
+      hideBranding: currentOrg?.hideBranding,
+    },
+  });
+
+  const isAdmin =
+    currentOrg &&
+    (currentOrg.user.role === MembershipRole.OWNER || currentOrg.user.role === MembershipRole.ADMIN);
+
+  if (isLoading) {
+    return <SkeletonLoader title={t("booking_appearance")} description={t("appearance_team_description")} />;
+  }
+  return (
+    <>
+      <Meta title={t("booking_appearance")} description={t("appearance_team_description")} />
+      {isAdmin ? (
+        <Form
+          form={form}
+          handleSubmit={(values) => {
+            mutation.mutate({
+              ...values,
+              theme: values.theme || null,
+            });
```

Comment:
We dont pass ID to orgs anymore - we get this from ctx in trpc 

as a user can only have one ORG per account